tests/Task.test.ts

```diff
@@ -77,6 +77,48 @@ describe('parsing', () => {
         expect(task!.tags[1]).toEqual('#journal/daily');
     });
 
+    it('parses a task from a line and extract tags and not global filter', () => {
+        // Arrange
+        const originalSettings = getSettings();
+        updateSettings({ globalFilter: '#task' });
+        const line =
+            '- [x] #task this is a done task #tagone #journal/daily 🗓 2021-09-12 ✅ 2021-06-20';
```

Comment:
I would trim out anything not relevant, so in this case, the dates.

(Testing of date-parsing is done in an earlier test)

Reply:
Covered as part of test refactor.